### ISSUE TITLE
rejectUnauthorized: false in check-started?

### ISSUE THREAD

**Issue by jreichenberg** [Polymer/web-component-tester#143]
This is likely pilot error, but strange behavior for me getting selenium started up via WCT.  When I run WCT against seed-element it all works.  If I set the `--root ./` option (see why below), I can't get past selenium startup: /wd/hub/status fails to resolve in check-started.js -- requests to it error out from Node TLS with 'UNABLE_TO_VERIFY_LEAF_SIGNATURE'.  If I add rejectUnauthorized: false to the require('request').defaults it gets past this and gives me a status code 504:
```
var request = require('request').defaults({json: true, rejectUnauthorized: false});''
```

If I further change getSeleniumHub to return a host of '0.0.0.0' instead of localhost, it all then works and tests proceed.
```
return 'http://0.0.0.0:' + port + '/wd/hub/status';
```

I know that the default value for rejectUnauthorized was changed in Node 0.10 from false to true, but I doubt that's the cause.

tl;dr

Here's why we're setting `--root ./` ...

seed-element sets .bowerrc to install dependencies in `../`. This works as a solution to the variability of bower dependency locations when running in a component repo vs. 'downstream' as part of another component or app but it scatters dependencies outside the repo you're working in. Our alternate solution is to leave the bower default directory and keep dependency paths as if they were being consumed 'downstream', but when running locally set an alternate root in the web server being used, e.g. ./bower_components.  The webserver will check this alt root before 404-ing. This is easy in both http-server (proxy option) and expressjs, and comes more natural to our devs as they are also working with LESS and SASS, which has a somewhat similar concept for handling variability of @import paths (see https://github.com/sass/node-sass#includepaths and https://github.com/gruntjs/grunt-contrib-less#paths)

**Comment by nevir** [comment]
Responding to the second half of your issue first :P

Yeah, the sibling dependency convention is deeply unsatisfying for app projects. However, you shouldn't _have_ to set root to the current directory when switching your bower directory to something more sane. WCT will still serve and detect your tests relative to where you ran it from (or the config); so it should be working regardless of where your components live?.

Downside, of course, is that it's potentially serving a bunch of things unrelated to your project (and a potential security issue if you are exposing those tests externally). But hopefully lets you run tests for now...

---

As for the real issue, this'll require some more digging.

**Comment by nevir** [comment]
I wasn't able to get much further with your selenium issue. However you should no longer need to do your `root` shenanigans on WCT master (and, soon, `3.0`). WCT now respects `bower_components` when present
